logger/LoggerThread.h

```diff
@@ -31,7 +31,7 @@ class LoggerThread
 
 private:
     void run();
-    void formatTime(const LogItem& item, char* output);
+    static void formatTime(const std::chrono::system_clock::time_point timestamp, char* output);
```

Comment:
private static methods are good candidates to be placed in anonymous namespaces

Reply:
yes, it was supposed to be public so it can be used outside. Fixed that.